packages/mux-player/src/template.ts

```diff
@@ -44,14 +45,18 @@ export const template = (props: MuxTemplateProps) => html`
       debug="${props.debug}"
       prefer-mse="${props.preferMse}"
       start-time="${props.startTime != null ? props.startTime : false}"
+      src="${!!props.src
```

Comment:
non-blocking: Any reason we aren't using the supported `playback-id` with token that supported in `mux-video`?

Reply:
yes, it's easier to write with also supporting setting the `src` attribute.
it was also a small issue that mux-video supports adding the token to the playback-id and mux-player does not.